.github/workflows/test-python.yaml

```diff
@@ -21,3 +21,16 @@ jobs:
           version: 23.9.1
           options: --check --exclude '/*kubeflow_org_v1*|__init__.py|api_client.py|configuration.py|exceptions.py|rest.py'
           src: sdk/
+
+      - name: Install dependencies
+        run: pip install pytest pytest-md pytest-emoji python-dateutil urllib3 kubernetes
+
+      - name: Run unit test for training sdk
+        uses: pavelzw/pytest-action@v2
```

Comment:
I don't think we should use this action since we should avoid third-party dependencies as much as possible.
Especially, since it is just a line simple script to run `pytest`, I don't think that using this script would be worth it.

Reply:
I have removed it.